src/containers/BankApi/BankApiInvalidBlocks/index.tsx

```diff
@@ -23,18 +23,48 @@ const BankApiInvalidBlocks: FC = () => {
       </p>
 
       <DocEndpoint endpoint="/invalid_blocks" method="GET" />
+      <TableParams
+        items={[
+          {
+            dataType: 'URL parameter',
+            description: 'If you want to start at a specific point, an offset can be specified.',
+            param: 'offset',
+          },
+          {
+            dataType: 'URL parameter',
+            description: 'Maximum number of block returned by the node. (max: 100)',
+            param: 'limit',
+          },
+        ]}
+      />
       <RequestResponseSnippet
-        code={`[
-  {
-    "id": "2bcd53c5-19f9-4226-ab04-3dfb17c3a1fe",
-    "created_date": "2020-07-11T18:44:16.518695Z",
-    "modified_date": "2020-07-11T18:44:16.518719Z",
-    "block_identifier": "65ae26192dfb9ec41f88c6d582b374a9b42ab58833e1612452d7a8f685dcd4d5",
-    "block": "3ff4ebb0-2b3d-429b-ba90-08133fcdee4e",
-    "confirmation_validator": "fcd2dce8-9e4f-4bf1-8dac-cdbaf64e5ce8",
-    "primary_validator": "51461a75-dd8d-4133-81f4-543a3b054149"
-  }
-]`}
+        code={`{
+  "count": 234,
+  "next": "http://143.110.137.54/invalid_blocks?limit=50&offset=50",
+  "previous": null,
+  "results": 
+  [
+    {
+      "id": "2bcd53c5-19f9-4226-ab04-3dfb17c3a1fe",
+      "created_date": "2020-07-11T18:44:16.518695Z",
+      "modified_date": "2020-07-11T18:44:16.518719Z",
+      "block_identifier": "65ae26192dfb9ec41f88c6d582b374a9b42ab58833e1612452d7a8f685dcd4d5",
+      "block": "3ff4ebb0-2b3d-429b-ba90-08133fcdee4e",
+      "confirmation_validator": "fcd2dce8-9e4f-4bf1-8dac-cdbaf64e5ce8",
+      "primary_validator": "51461a75-dd8d-4133-81f4-543a3b054149"
+    },
+    ...
```

Comment:
let's keep just 2 records here and remove this `...` like mentioned in the description 2 should be enough for listing records. also, it will be consistent with other devs and docs.

Reply:
Alright, done. I also changed offset value to 2 to keep it consistent.